src/ctap/apdu.rs

```diff
@@ -168,12 +168,17 @@ impl TryFrom<&[u8]> for APDU {
         }
         if payload.len() > 2 {
             // Lc is possibly three-bytes long
-            let extended_apdu_lc: usize = BigEndian::read_u16(&payload[1..]) as usize;
-            let extended_apdu_le_len: usize = if payload.len() > extended_apdu_lc {
-                payload.len() - extended_apdu_lc - 3
-            } else {
-                0
+            let extended_apdu_lc: usize = BigEndian::read_u16(&payload[1..3]) as usize;
```

Comment:
Do you need to make the type explicit, if you already assign `as usize`?

Reply:
yup it wasn't needed, removed